source/_integrations/refoss.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Refoss
+description: Integrate Refoss devices
+ha_category:
+  - Switch
+ha_release: 2023.11
+ha_codeowners:
+  - '@ashionky'
+ha_iot_class: Local Polling
+ha_domain: refoss
+featured: true
+ha_config_flow: true
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+Integrate Refoss devices into Home Assistant.
+
+{% include integrations/config_flow.md %}
```

Comment:
It seems like there are Prerequisites that need to be handled by a user before they can actually configure it...

So the documentation should be structured like that.

Start with the prerequisites and show the config flow after that.